rinex/tests/sampling.rs

```diff
@@ -23,25 +24,119 @@ mod sampling {
     }
     #[test]
     fn decimate_by_interval() {
-        let path = env!("CARGO_MANIFEST_DIR").to_owned()
-            + "/../test_resources/NAV/V3/AMEL00NLD_R_20210010000_01D_MN.rnx";
-        let mut rinex = Rinex::from_file(&path).unwrap();
-        let initial_epochs = rinex.epochs();
+        let mut rinex =
```

Comment:
I'd keep using `CARGO_MANIFEST_DIR` here, it's nice to be able to run the tests from anywhere in the project.